app/assets/stylesheets/components/_card-auth.scss

```
@@ -0,0 +1,21 @@
+.card-auth {
+  @extend .card;
+
+  width: 45%;
+  margin: auto;
+  margin-top: 30px;
```

We can leverage the functions from `'functions/sizing'`. We always practice to use `rem`.
```suggestion
  margin-top: rem(30px);
```

---

Resolved in a89ff14